public/v4/apps/lemmy.yml

```diff
@@ -0,0 +1,146 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: postgres:15-alpine
+        environment:
+            POSTGRES_USER: lemmy
+            POSTGRES_PASSWORD: $$cap_db_pass
+            POSTGRES_DB: lemmy
+        volumes:
+            - $$cap_appname-db-data:/var/lib/postgresql/data
+        restart: always
+        caproverExtra:
+            notExposeAsWebApp: true
+
+    $$cap_appname-pictrs:
+        image: asonix/pictrs:0.5.0
+        user: '991:991'
```

Comment:
This is not consumed by CapRover. I assume it's not important and it works without this?